ui/css/design-system/_colors.scss

```diff
@@ -24,13 +24,11 @@ $color-map: (
   'warning-muted': --color-warning-muted,
   'warning-inverse': --color-warning-inverse,
   'success-default': --color-success-default,
-  'success-alternative': --color-success-alternative,
```

Comment:
Must have been a bad merge `success-alternative` no longer exists in the codebase

![Screenshot 2024-06-07 at 4 06 48 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/36261952-8ef2-43d7-9b8a-743dc7f97e63)